app/Http/Controllers/InvoiceController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Imports\Import;
+use App\Models\Invoice;
+use Illuminate\Support\Facades\Log;
+
+
+class InvoiceController extends Controller
+{
+    public function index()
+    {
+        $invoice = Invoice::with('item')->paginate(15);
+        return response()->json($invoice);
+    }
+
+    public function test($name)
+    {
+        $score = random_int(0,100);
+        return response()->json(['name' => $name,'score' => $score]);
+    }
+
+    public function show(int $id)
+    {
+        $invoice = Invoice::with('item')->find($id);
+
+        if (!empty($invoice))
```

Comment:
use early exit pattern by swap the if else order, this would avoid the else part completely